src/shims/unix/linux/epoll.rs

```diff
@@ -272,6 +272,8 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
     ) -> InterpResult<'tcx, Scalar> {
         let this = self.eval_context_mut();
 
+        this.check_no_isolation("`epoll_wait`")?;
```

Comment:
Add to the message that the order of results depends on file descriptors and hypothetically on interpreter pointer addresses (only when reusing integer file descriptors after closing a previous one)